SU-SWS/stanford_profile
# D8CORE-1279: Fix for subtitle/subhead styles

# Maybe READY FOR REVIEW - advice needed
- (Edit the above to reflect status)

# Summary
- TL;DR - Fixes the styles for the subtitle/subhead

# Needed By (Date)
- Before sprint end

# Urgency
- Would be needed for any site using these styles

# Steps to Test

1. Check out this branch
1. Clear Cache
1. In the editor add "Subtitle" to text
1. Save.
1. Verify that the Subtitle text is:

![Screen Shot 2020-02-19 at 6 01 37 PM](https://user-images.githubusercontent.com/284440/74893723-e64ce480-5341-11ea-985f-e923019579b8.png)


# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

D8Core

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-1279
- Other PRs: This is taking over these PRs:
https://github.com/SU-SWS/stanford_profile/pull/109
https://github.com/SU-SWS/stanford_profile/pull/107

- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)

Please see https://stanfordits.atlassian.net/browse/D8CORE-1279?focusedCommentId=90556&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-90556 

- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
